### DISCUSSION TITLE
Is it possible to run BlockSparseAttention with cuda 11.0-11.2?

### DISCUSSION THREAD

**Discussion by fedorovgv** [facebookresearch/xformers#648] [General]
I mean I want to use [block sparse attention](https://github.com/facebookresearch/xformers/blob/6f3c20f223c0f051e4b2dc744cc7591591ee04c4/xformers/components/attention/blocksparse.py#L46) which uses triton which requires cuda 11.4. And maybe there is a previous version of xformer using triton that requires cuda of a smaller version? I need cuda 11.0 - 11.1 (max 11.2) due to server restrictions. 

Now i have `RuntimeError: Triton requires CUDA 11.4+` with cuda 11.0.

**Comment by danthe3rd** [comment]
I would recommend to use CUDA at least 11.6, and we don't support < 11.3. If you can make it work, we would gladly accept a PR, but otherwise it's not something we will fix.
Regarding this specific error, it's more [a problem of triton](https://github.com/openai/triton), so I suggest asking there.
